tests/models/falcon/test_modeling_falcon.py

```diff
@@ -438,6 +443,65 @@ def test_model_rope_scaling(self, scaling_type):
         # The output should be different for long inputs
         self.assertFalse(torch.allclose(original_long_output, scaled_long_output, atol=1e-5))
 
+    def test_model_rope_scaling(self):
```

Comment:
This test is the same on all models, with two variations:
- Llama passes `position_ids` to RoPE, as opposed to an integer depicting the sequence length
- GPTNeoX has `base=config.rotary_emb_base,`, all other models have `base=config.rope_theta`. This parameter is the same in GPTNeoX, but it has a different name

Reply:
In this case, we can employ some `Copied from`s :)  

I don't think there's an easy way for the llama test - but for GPTNeox remapping with `with x->y` should work